packages/titus-backend/Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM node:14-alpine as builder
+FROM node:lts-alpine@sha256:fb6cb918cc72869bd625940f42a7d8ae035c4e786d08187b94e8b91c6a534dfd as builder
```

Comment:
this is the only one part that I don't necessarily agree with. as this is an accelerator, it is supposed to be future proof. using whatever node 14 lts is the latest is a good way to ensure that you get... the latest 14 lts, which is stable enough and especially may contain security fixes which are not in the revision that you pinned. let's restore it to 14-alpine

Reply:
Makes sense, this is an accelerator, not a final product.